lib/sles4sap.pm

```diff
@@ -209,18 +209,23 @@ sub copy_media {
     my ($self, $proto, $path, $nettout, $target) = @_;
 
     # First copy media
+    my $mnt_path = '/mnt';
     assert_script_run "mkdir $target";
-    assert_script_run "mount -t $proto $path /mnt";
-    type_string "cd /mnt\n";
+    assert_script_run "mount -t $proto -o ro $path $mnt_path";
+    type_string "cd $mnt_path\n";
     type_string "cd " . get_var('ARCH') . "\n";    # Change to ARCH specific subdir if exists
     assert_script_run "cp -ax . $target/", $nettout;
 
-    # Then verify everything was copied correctly
-    my $cmd = q|find . -type f -exec md5sum {} \; > /tmp/check-nw-media|;
-    assert_script_run $cmd, $nettout;
+    # Go back to target directory and umount the share, as we don't need it anymore
     type_string "cd $target\n";
-    assert_script_run "umount /mnt";
-    assert_script_run "md5sum -c /tmp/check-nw-media", $nettout;
+    assert_script_run "umount $mnt_path";
+
+    # Then verify everything was copied correctly
+    # NOTE: checksum is generated with this command: "find . -type f -exec md5sum {} \; > checksums.md5sum"
+    my $chksum_file = 'checksum.md5sum';
+    # We can't check the checksum file itself as well as the clustered NFS share part
+    assert_script_run "sed -i -e '/$chksum_file\$/d' -e '/\\/nfs_share/d' $chksum_file";
+    assert_script_run "md5sum -c $chksum_file", $nettout;
```

Comment:
```suggestion
    assert_script_run "md5sum --quiet -c $chksum_file", $nettout;
```

Reply:
I will merge this and then you guys decided whether to add the `--quiet` or not later. I vote no, BTW. Better to have the output if this step fails to quickly pinpoint which file is failing the check.

Reply:
> I will merge this and then you guys decided whether to add the `--quiet` or not later. I vote no, BTW. Better to have the output if this step fails to quickly pinpoint which file is failing the check.

`md5sum` doesn't exit earlier if a file doesn't pass MD5.  The `--quiet` output is used to avoid useless output.

Reply:
I tested and yes `--quiet` is a good option, and better than the default behavior, because when I tested it I had only the failing files instead of the whole list:
```
# md5sum -c --quiet checksums.md5sum
./openqa.suse.de.crt: FAILED
./SUSE_CA_suse.de.chain.crt: FAILED
md5sum: WARNING: 2 computed checksums did NOT match
```

So easier to analyze!

I will do a new PR for this.